phpunit/functional/Glpi/Form/Condition/EngineTest.php

```diff
@@ -412,10 +412,67 @@ public static function firstSectionShouldAlwaysBeVisible(): iterable
         ];
     }
 
+    public static function conditionsOnQuestionWithNullExtraData(): iterable
+    {
+        $form = new FormBuilder();
```

Comment:
data provider should not change anything in database, this cause issues with phpunit

Reply:
The `FormBuilder` class only store parameters, it does not create any items.